Makefile

```diff
@@ -18,12 +18,12 @@ default: help
 
 VERSION ?= 1.6.0
 
-# 2.15 image: "2.15.0-alpine"
-TARGET_APISIX_VERSION ?= "3.1.0-centos"
+
+TARGET_APISIX_VERSION ?= "dev"
 APISIX_ADMIN_API_VERSION ?= "v3"
 ifneq ($(APISIX_ADMIN_API_VERSION), "v3")
-ifeq ($(TARGET_APISIX_VERSION), "3.1.0-centos")
-	TARGET_APISIX_VERSION = "2.15.0-alpine"
+ifeq ($(TARGET_APISIX_VERSION), "dev")
+	TARGET_APISIX_VERSION = "3.1.0-centos"
```

Comment:
This is incorrect, we need to keep tests for v2, v3 and dev.

Reply:
TARGET_APISIX_VERSION ?= "dev"
APISIX_ADMIN_API_VERSION ?= "v3"

ifeq ($(APISIX_ADMIN_API_VERSION), "v3")
    ifeq ($(TARGET_APISIX_VERSION), "dev")
        TARGET_APISIX_VERSION = "3.1.0-centos"
    endif
else ifeq ($(APISIX_ADMIN_API_VERSION), "v2")
    ifeq ($(TARGET_APISIX_VERSION), "dev")
        TARGET_APISIX_VERSION = "2.12-centos"
    endif
else
    ifeq ($(TARGET_APISIX_VERSION), "dev")
        TARGET_APISIX_VERSION = "2.12-centos"
    endif
endif



What should be in the condition like ifeq ($(APISIX_ADMIN_API_VERSION), "v3") is true and ifeq ($(TARGET_APISIX_VERSION), "dev")
then I should:


TARGET_APISIX_VERSION = "3.1.0-centos"

?

Reply:
2.15.3-centos
v2

3.2.0-centos
v3

dev
dev

Is above correct?